### ISSUE TITLE
Failed to update beatmap database.

### ISSUE THREAD

**Issue by itdelatrisu** [fluddokt/opsu#77]
On version 0.13.0a, only when first launching the app with no databases previously created.
```
Error: Failed to update beatmap database. 
java.sql.SQLException: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: SELECT value FROM info WHERE key = 'version'
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:430)
	at org.sqldroid.SQLDroidConnection.chainException(SQLDroidConnection.java:158)
	at org.sqldroid.SQLiteDatabase.rawQuery(SQLiteDatabase.java:124)
	at org.sqldroid.SQLDroidStatement.executeQuery(SQLDroidStatement.java:165)
	at itdelatrisu.opsu.db.BeatmapDB.updateDatabase(BeatmapDB.java:238)
	at itdelatrisu.opsu.db.BeatmapDB.init(BeatmapDB.java:137)
	at itdelatrisu.opsu.db.DBController.init(DBController.java:49)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:199)
	at fluddokt.opsu.fake.GameOpsu.render(GameOpsu.java:92)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:422)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1548)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1259)
Caused by: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: SELECT value FROM info WHERE key = 'version'
	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:889)
	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:500)
	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
	at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
	at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:44)
	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1318)
	at android.database.sqlite.SQLiteDatabase.rawQuery(SQLiteDatabase.java:1257)
	at org.sqldroid.SQLiteDatabase.rawQuery(SQLiteDatabase.java:117)
	... 9 more
```

**Comment by flazep3** [comment]
Yeah i got that too

**Comment by fluddokt** [comment]
Hopefully fixed in 0.14.0a
